components/HowToPanel.tsx

```diff
@@ -4,23 +4,32 @@ import { IconSetContext } from '../utils/IconSetContext'
 import Button from './Button'
 import IconEditor from './IconEditor'
 import toast, { Toaster } from 'react-hot-toast'
+import { HowToPanelProps } from '../interface'
 
-const HowToPanel = (props) => {
-  const { show, close, iconName, type, iconTags } = props
-
+const HowToPanel: React.FC<HowToPanelProps> = ({
+  show,
+  close,
+  iconName,
+  type,
+  iconTags
+}) => {
   const { iconDispatch } = useContext(IconSetContext)
-  const ref = useRef()
+  const ref = useRef<HTMLDivElement>(null)
   const [iconEditor, setIconEditor] = useState(false)
   const [iconType, setIconType] = useState('static')
```

Comment:
We can't use all strings here, right? A better alternative:
`type IconType = "static" | "dynamic"`